app/src/main/res/layout/fragment_home.xml

```diff
@@ -5,24 +5,50 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <TextView
-        android:id="@+id/vHomeWelcomeTextView"
+    <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/spacing_huge"
-        android:gravity="center"
-        android:text="@string/fragment_home_welcome_text_view"
-        app:layout_constraintTop_toTopOf="parent" />
-
-    <Button
-        android:id="@+id/vHomeLogoutButton"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="@dimen/spacing_huge_extra"
-        android:gravity="center"
-        android:text="@string/fragment_home_logout_button"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"/>
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <android.support.constraint.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/fragment_home_toolbar_height"
+            android:elevation="@dimen/spacing_large">
+
+            <ImageView
+                android:id="@+id/vHomeBarLogo"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:src="@drawable/wolox_logo"
+                android:layout_marginStart="@dimen/spacing_large"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent" />
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/fragment_home_wolox_toolbar"
```

Comment:
Be careful with attributes order. Layout dimens modifications first and then text/src/etc attributes

Reply:
✔ **Done**